### PR TITLE
test(editor): Add more workflow actions tests

### PR DESCRIPTION
**Pull request by MiloradFilipovic**

### CODE REVIEW

**Review by ivov — DISMISSED** [review]

File: cypress/e2e/7-workflow-actions.cy.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	it('should copy nodes', () => {
+		WorkflowPage.actions.addNodeToCanvas(SCHEDULE_TRIGGER_NODE_NAME);
+		WorkflowPage.actions.addNodeToCanvas(CODE_NODE);
+		cy.get('body').type('{meta}', { release: false }).type('a');
+		cy.get('body').type('{meta}', { release: false }).type('c');
+		WorkflowPage.getters.successToast().should('exist');
+	});
+
+	it('should paste nodes', () => {
+		cy.get('body').paste(`{"meta":{"instanceId":"1a30c82b98a30444ad25bce513655a5e02be772d361403542c23172be6062f04"},"nodes":[{"parameters":{"rule":{"interval":[{}]}},"id":"a898563b-d2a4-4b15-a979-366872e801b0","name":"Schedule Trigger","type":"n8n-nodes-base.scheduleTrigger","typeVersion":1,"position":[420,260]},{"parameters":{"options":{}},"id":"b9a13e3d-bfa5-4873-959f-fd3d67e380d9","name":"Set","type":"n8n-nodes-base.set","typeVersion":1,"position":[640,260]}],"connections":{"Schedule Trigger":{"main":[[{"node":"Set","type":"main","index":0}]]}}}`);
```
Comment (ivov) [inline cypress/e2e/7-workflow-actions.cy.ts:101]: Does paste content belong in fixtures?

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -128,3 +128,16 @@ Cypress.Commands.add('resetAll', () => {
...
+
+Cypress.Commands.add('paste', { prevSubject: true }, (selector, pastePayload) => {
+	// https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event
+	cy.wrap(selector).then($destination => {
+		const pasteEvent = Object.assign(new Event('paste', { bubbles: true, cancelable: true }), {
+		clipboardData: {
+			getData: () => pastePayload
+		}
+		});
+		console.log(pastePayload);
```
Comment (ivov) [inline cypress/support/commands.ts:140]:
```suggestion

```

**Review by ivov — DISMISSED** [review]
This suite passes but for reference these other suites fail locally in headless mode:
- `2-credentials.cy.ts`
- `5-ndv.cy.ts`
- `9-expression-editor-modal.cy.ts`

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.206.1`

**Pull request merged**